pkgs/os-specific/linux/spl/default.nix

```diff
@@ -25,7 +26,8 @@ let
 
       patches = [ ./const.patch ./install_prefix.patch ];
 
-      nativeBuildInputs = [ autoreconfHook ];
+      nativeBuildInputs = [ autoreconfHook ]
+        ++ optional (stdenv.lib.versionAtLeast kernel.version "4.14") libelf;
```

Comment:
E.g. this would become just 
````
nativeBuildInputs = [ autoreconfHook ] ++ kernel.moduleBuildDependencies;
````

Reply:
sure, it could be cleaner.

Reply:
@dguibert Are you working on this? I'd like to fix this asap and would do it myself otherwise.

Reply:
yes, I'll update this branch very soon.

Reply:
done.